web/management/commands/create_test_data.py

```
@@ -575,3 +637,280 @@ def handle(self, *args, **kwargs):
+    def generate_mock_answers(self, quiz: Quiz) -> dict:
+        """Generate mock answers for a quiz submission."""
+        answers = {}
+        questions = quiz.questions.all()
+
+        for question in questions:
+            q_id = str(question.id)
+
+            if question.question_type == "multiple":
+                correct_option = question.options.filter(is_correct=True).first()
+                # 70% chance of choosing the correct answer
+                if random.random() < 0.7:
```

Ruff (0.8.2)

Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)